app/src/main/java/com/github/se/signify/ui/screens/challenge/CreateAChallengeScreen.kt

```diff
@@ -230,3 +234,11 @@ fun ModeButton(
           else MaterialTheme.colorScheme.onSurface,
   )
 }
+
+fun getWordsForMode(context: Context, mode: ChallengeMode): List<String> {
```

Comment:
This function is well-implemented, using Kotlin's standard library to create concise and effective logic for word selection.